examples/Demo/FluentUI.Demo.Client/Documentation/Components/Slider/FluentSlider.md

```diff
@@ -26,20 +26,42 @@ The **FluentSlider** component is an input where the user selects a value from w
 
 View the [Usage Guidance](https://fluent2.microsoft.design/components/web/react/slider/usage).
 
+## Default
+
+By default, the slider is a horizontal slider with a range of `0` to `100` and a step of `1`.
+
 {{ SliderDefault }}
 
-## Label
+## Min, Max and Step
+
+You can define the `Step` value of a slider so that the value will always be a mutiple of that step.
+
+{{ SliderMinMaxStep }}
+
+## ReadOnly and Disabled
+
+The slider can be set to `ReadOnly` or `Disabled` mode.
 
-{{ SliderLabel }}
+{{ SliderReadOnlyDisabled }}
+
+> The **FluentSlider** component does not support the `ReadOnly` attribute (using the keyboard),
+> like the default [input type="range"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) HTML element.
+> The `ReadOnly` attribute is not applicable to the slider because it is designed for text-based inputs,
+> and sliders are primarily used for selecting a value within a range
+>
+> The `Disabled` attribute is used to prevent user interaction with the slider, while `ReadOnly` would imply
+> that the slider can still be interacted with but not modified, which doesn't align with the intended use of a slider.
```

Comment:
> The **FluentSlider** component does not support the `ReadOnly` attribute. This is in line with the HTML standard that prescribes only inputs that accept text entry may be readonly. See  the [HTML spec](https://html.spec.whatwg.org/multipage/input.html#the-readonly-attribute) and the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) for more information.
>
> The `Disabled` attribute is used to prevent user interaction with the slider.